src/main/java/db/migration/V227__Migrate_TeachingCallReceipts_Comments_To_TeachingCallComments.java

```diff
@@ -0,0 +1,70 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+public class V227__Migrate_TeachingCallReceipts_Comments_To_TeachingCallComments implements JdbcMigration {
+    @Override
+    public void migrate(Connection connection) throws Exception {
+        // Create TeachingCallComments table
+        String createTeachingCallComments = "CREATE TABLE IF NOT EXISTS `TeachingCallComments` ("
+                + "`Id` INT(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,"
+                + "`Comment` TEXT NOT NULL,"
+                + "`TeachingCallReceiptId` INT(11),"
+                + "`UpdatedAt` TIMESTAMP NULL,"
+                + "`CreatedAt` TIMESTAMP NULL,"
+                + "`ModifiedBy` VARCHAR(16) NULL,"
+                + "FOREIGN KEY (`TeachingCallReceiptId`)"
+                + " REFERENCES `TeachingCallReceipts`(`id`));";
+
+        PreparedStatement createTableStatement = connection.prepareStatement(createTeachingCallComments);
+
+        // Get TeachingCallReceipts comments
+        PreparedStatement commentsQuery = connection.prepareStatement(
+            "SELECT"
+                    + " TeachingCallReceipts.`Id` AS TeachingCallReceiptId, "
+                    + " CONCAT(`firstName`, ' ', `lastName`) AS Author, "
```

Comment:
Do you use this Author column anywhere? It looks like its calculated, but then not used?

Reply:
I think I calculated it at first but didn't end up saving it since the Author is just the instructor filling out the Teaching Call form. It didn't seem necessary to display the name.